products/neo4j.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Neo4j
+permalink: /neo4j
+releasePolicyLink: https://neo4j.com/terms/support-terms/
+category: db
+changelogTemplate: https://github.com/neo4j/neo4j/tags/__LATEST__/
```

Comment:
Links produced by this changelogTemplate are broken (for instance: https://github.com/neo4j/neo4j/tags/5.2/, https://github.com/neo4j/neo4j/tags/4.4/).

Reply:
Yes, indeed... and I don't know what to do about it :sweat:

Reply:
I have an idea, I'm working on it

Reply:
Dedicated issue :crossed_fingers:  : 

- https://github.com/neo4j/neo4j/issues/13012